implementations/elixir/ockam/ockam/test/ockam/secure_channel_test.exs

```diff
@@ -389,14 +392,62 @@ defmodule Ockam.SecureChannel.Tests do
     {:ok, channel} = create_secure_channel([listener], bob, %{bar: :foo})
 
     {:ok, me} = Ockam.Node.register_random_address()
-    Ockam.Router.route("PING!", [channel, me], [me])
+    Worker.route("PING!", [channel, me], [me], %{}, nil)
 
     refute_receive %Ockam.Message{
       onward_route: [^me],
       payload: "PING!"
     }
   end
 
+  test "dynamic identity" do
```

Comment:
this looks strange,  these tests where removed some time ago,  and should be failing now if they ran due to changes on the API.
So need to remove these,   probably they come from some conflicts during a merge/rebase on your branch?

Reply:
I am also not sure about why these are here if they were removed. I was keeping my branch up to date with Github's merge function. I'll review and remove these.

Reply:
I've removed the old test in commit https://github.com/build-trust/ockam/commit/a6e3faeece3e6c6f3d2dee05d9269e9ea9152151.